platform/android/java/editor/build.gradle

```diff
@@ -45,16 +45,16 @@ android {
         }
 
         debug {
-            initWith release
-
             // Need to swap with the release signing config when this is ready for public release.
             signingConfig signingConfigs.debug
```

Comment:
@Calinou You can remove the `signingConfig signingConfigs.debug` line (and the comment that goes with it) as this is already the default configuration.